piker/ui/_exec.py

```diff
@@ -114,6 +145,32 @@ def __init__(self, parent=None):
         self.setMinimumSize(*self.size)
         self.setWindowTitle(self.title)
 
+        self._status_bar: QStatusBar = None
+        self._status_label: QLabel = None
+
+    @property
+    def mode_label(self) -> QtGui.QLabel:
+
+        # init mode label
+        if not self._status_label:
+            # TODO: i guess refactor stuff to avoid having to import here?
+            from ._style import _font_small, hcolor
```

Comment:
Hmm i wonder if it's worth it to just now move all this stuff to a new `._window.py` mod?

I'm thinking it'll be a nice cleanup.

Reply:
Did this.